build.sbt

```diff
@@ -1253,6 +1254,20 @@ lazy val nettyServerCats: ProjectMatrix = (projectMatrix in file("server/netty-s
   .jvmPlatform(scalaVersions = scala2And3Versions)
   .dependsOn(serverCore, nettyServer, cats, serverTests % Test)
 
+lazy val nettyServerZio: ProjectMatrix = (projectMatrix in file("server/netty-server/zio"))
+  .settings(commonJvmSettings)
+  .settings(
+    name := "tapir-netty-server-zio",
+    libraryDependencies ++= Seq(
+      "io.netty" % "netty-all" % "4.1.82.Final",
+      "dev.zio" %% "zio-interop-cats" % Versions.zioInteropCats,
+    ) ++ loggerDependencies,
+    // needed because of https://github.com/coursier/coursier/issues/2016
+    useCoursier := false
```

Comment:
maybe some common netty-settings to avoid duplication?

Reply:
I think that not much things can be move to common as most setting is different between backends

Reply:
even if it's 2 settings let's factor them out so that we don't repeat the comments etc.